knowledge_base/ADRs/001-Hexagonal-Architecture-Style.md

```diff
@@ -0,0 +1,44 @@
+# ADR 001: Hexagonal Architecture Style
+
+Date: 2024-01-12
```

Comment:
Is a datestamp here necessary? Nitpicking only because whatever format we establish here will become a template for all future ADRs. If we have a `Proposed` and `Accepted` datestamp immediately below, what additional information does this provide?